backend/finalBackendServer.js

```diff
@@ -0,0 +1,444 @@
+var express = require('express');
+var app = express();
+
+const bodyParser = require('body-parser');
+const { application } = require('express');
+
+var util = require('util');
+var encoder = new util.TextEncoder('utf-8');
+const { MongoClient } = require("mongodb");
+const { response } = require("express");
+const uri = "mongodb://127.0.0.1:27017"
+const client = new MongoClient(uri)
+
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+/**
+ * purpose: get a specific detailed look of a post
+ */
+
+
+ app.post('/specific', async(req, res)=>{
+    var type = req.body.facility_type;
+    console.log(type);
+    var numberOfType = parseInt(type);
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `··`](https://app.codacy.com/gh/HizanLi/CPEN321/pullRequest?prid=9816566)
```suggestion
  var numberOfType = parseInt(type);
```